packages/amplify_storage_s3/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/types/FlutterGetUrlRequest.kt

```diff
@@ -48,8 +49,11 @@ data class FlutterGetUrlRequest(val request: Map<String, *>) {
     }
 
     companion object {
-        fun isValid(request: Map<String, *>): Boolean {
-            return request["key"] is String
+        private const val validationErrorMessage: String = "GetUrl request malformed."
+        fun validate(request: Map<String, *>) {
+            if(request["key"] !is String? || request["key"] == null) {
+                throw AmplifyFlutterValidationException(validationErrorMessage, "Ensure that key variable is present.")
```

Comment:
Since a number of these messages are used in multiple validation functions, could you make create constants for them in a `ValidationMessages.kt` and reuse them wherever needed?

Same in other validation functions.

Reply:
Yeah, I wanted to investigate the best way to do that with template strings.

Reply:
Fixed.